console/backend/src/main/java/org/frankframework/management/web/AdapterStatistics.java

```diff
@@ -28,7 +28,7 @@
 @RestController
 public class AdapterStatistics extends FrankApiBase {
 
-	@RolesAllowed({"IbisObserver", "IbisDataAdmin", "IbisAdmin", "IbisTester"})
+	@AllRolesAllowed
```

Comment:
Dit is niet goed. De IbisWebService role mag hier niet bij.

Reply:
Maar die staat ook niet in de `AllRolesAllowed`. Zo te zien wordt `IbisWebService` niet gebruikt in deze code. Wellicht is `AllRolesAllowed` dan niet de juiste naam?

Reply:
Ok, duidelijk. Dit is een eigen annotatie. Werkt dit echt? En zo ja, dan moet hij wel een goede naam krijgen. Want nu is het dus niet alles maar alle user rollen. Overigens had je de annotations ook kunnen vervangen door `@RolesAllowed(DynamicRegistration.ALL_IBIS_USER_ROLES)`

Reply:
Dat lijkt heel logisch maar dat werkt niet: `Attribute value must be constant` - compileert niet.